src/bundle/Resources/views/locationview.html.twig

```diff
@@ -21,21 +21,26 @@
             <div class="ez-tabs-panel is-tab-selected" id="ez-tab-view">
                 {{ render(controller('ez_content:viewAction', {'contentId': contentId, 'locationId': locationId, 'viewType': 'content_tab'})) }}
             </div>
-            <div class="ez-tabs-panel" id="ez-tab-details">
-                {{ render(controller('ez_content:viewAction', {'contentId': contentId, 'locationId': locationId, 'viewType': 'details_tab'})) }}
-            </div>
-            <div class="ez-tabs-panel" id="ez-tab-versions">
-                {{ render(controller('ez_content:viewAction', {'contentId': contentId, 'locationId': locationId, 'viewType': 'versions_tab'})) }}
-            </div>
-            <div class="ez-tabs-panel" id="ez-tab-locations">
-                {{ render(controller('ez_content:viewAction', {'contentId': contentId, 'locationId': locationId, 'viewType': 'locations_tab'})) }}
-            </div>
-            <div class="ez-tabs-panel" id="ez-tab-relations">
-                {{ render(controller('ez_content:viewAction', {'contentId': contentId, 'locationId': locationId,'viewType': 'relations_tab'})) }}
-            </div>
-            <div class="ez-tabs-panel" id="ez-tab-translations">
-                {{ render(controller('ez_content:viewAction', {'contentId': contentId, 'locationId': locationId, 'viewType': 'translations_tab'})) }}
-            </div>
+            <ez-asynchronous-block
+                class="ez-tabs-panel" id="ez-tab-details"
+                url="{{ path('_ez_content_view', {'contentId': contentId, 'locationId': locationId, 'viewType': 'details_tab'}) }}"
```

Comment:
Opened https://github.com/ezsystems/ezpublish-kernel/pull/2048 that would allow to simplify those links.